client/components/mma/switch/SwitchReview.stories.tsx

```diff
@@ -27,12 +31,52 @@ export default {
 export const Default: ComponentStory<typeof SwitchReview> = () => (
 	<SwitchReview />
 );
+// We're not setting `state` in the component-level parameters above due to how
```

Comment:
I'm torn on this one - not sure this comment will help us if we run into this issue in a different place. It seems like the problem is more with how we check for one payment method vs another. The data structure implies they are not mutually exclusive, while in practice they are. That said, the `subscription.card && <>foo</>` is a very neat little way to check it exists.

Reply:
The comment was more to explain how parameters are merged in stories which is not necessarily obvious and has caused confusion elsewhere. In this case it manifests itself as multiple payment methods being shown due to how we check the payment method, but, as you say, this wouldn't happen in practice.

I was hesitant to switch to using `paymentMethod` instead as I'm not sure how reliable that is as it doesn't seem to be used elsewhere in the codebase. It would be a neater solution than checking for properties associated with specific payment methods though.